public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -582,33 +584,68 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
     });
 
     // for new panel header design
+    const containerClassNamesNewPanelChrome = classNames({
+      'panel-container--absolute': isSoloRoute(locationService.getLocation().pathname),
+      'panel-container--transparent': transparent,
+      'panel-container--no-title': this.hasOverlayHeader(),
+      [`panel-alert-state--${alertState}`]: alertState !== undefined,
+    });
+
     const onCancelQuery = () => panel.getQueryRunner().cancelQuery();
     const title = panel.getDisplayTitle();
     const noPadding: PanelPadding = plugin.noPadding ? 'none' : 'md';
     const leftItems = [
       <PanelHeaderLoadingIndicator state={data.state} onClick={onCancelQuery} key="loading-indicator" />,
     ];
 
+    const titleItems = (innerWidth: number, innerHeight: number) => {
+      return (
+        <PanelHeaderTitleItems
+          key="title-items"
+          innerHeight={innerHeight}
+          innerWidth={innerWidth}
+          panelDescription={panel.description}
+          links={getPanelLinksSupplier(panel)}
+          scopedVars={panel.scopedVars}
+          replaceVariables={panel.replaceVariables}
+          alertState={alertState}
+          data={data}
+          panelId={panel.id}
+        />
+      );
+    };
     if (config.featureToggles.newPanelChromeUI) {
       return (
-        <PanelChrome width={width} height={height} title={title} leftItems={leftItems} padding={noPadding}>
-          {(innerWidth, innerHeight) => (
-            <>
-              <ErrorBoundary
-                dependencies={[data, plugin, panel.getOptions()]}
-                onError={this.onPanelError}
-                onRecover={this.onPanelErrorRecover}
-              >
-                {({ error }) => {
-                  if (error) {
-                    return null;
-                  }
-                  return this.renderPanelContent(innerWidth, innerHeight);
-                }}
-              </ErrorBoundary>
-            </>
-          )}
-        </PanelChrome>
+        <section
+          className={containerClassNamesNewPanelChrome}
+          aria-label={selectors.components.Panels.Panel.containerByTitle(panel.title)}
+        >
```

Comment:
this section should not be needed, aria-label can be moved to PanelChrome

Reply:
Yes, that makes sense, aria-label is now in PanelChrome. We'll remove the `section` [when we implement](https://github.com/grafana/grafana/pull/60514#discussion_r1067980557) better ways to handle _transparency_ and _embedded solo panel_ in a next PR.

Reply:
@polibb would just remove it in this PR as it adds no value in its current state, but leave it to you to decide

Reply:
> We'll remove the section https://github.com/grafana/grafana/pull/60514#discussion_r1067980557 better ways to handle transparency and embedded solo panel in a next PR.

'panel-container--absolute' and  'panel-container--transparent' have no impact on PanelChrome so just not sure why it was added this wrapper, these classes only impact the old table styles which the new PanelChrome don't use :)

Reply:
The new panel will not be able to be shared in an embedded form (absolute positioning) and will not have the transparent (background) option?

Reply:
@polibb of course it will, but just saying that those classes have no effect on this new PanelChrome so not sure why they wrap the new PanelChrome